src/main/java/ua/samoplavskaya/HW_hibernate/Main.java

```diff
@@ -0,0 +1,96 @@
+package ua.samoplavskaya.HW_hibernate;
+
+import jakarta.persistence.*;
+import org.hibernate.query.hql.spi.SemanticPathPart;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+
+public class Main {
+    public static void main(String[] args) {
+        EntityManagerFactory emf = Persistence.createEntityManagerFactory("MyUnit");
+        EntityManager entityManager = emf.createEntityManager();
+        entityManager.getTransaction().begin();
+        //task1(entityManager);
+
+        // User user = task2(entityManager, 5);
+        entityManager.getTransaction().commit();
+        entityManager.close();
+//        System.out.println("List of orders for user " + user.getFirstname() + " " + user.getLastname() + ": ");
+//        System.out.println(user.getOrders());
+        try (EntityManager entityManager1 = emf.createEntityManager()) {
+            entityManager1.getTransaction().begin();
+            task3(entityManager1);
+        }
+        emf.close();
+    }
+
+
+    public static void task1(EntityManager entityManager) {
```

Comment:
In each task, it would be nice, if you get entity manager and do operation. For example:

```
task1Save() {
    try(EntityManager manager = getEntityManager()) {
        ... your code
    }
}
```

getEntityManager -- private method that returns EntityManager by configured SessionFactory